src/status_im/biometric_auth/core.cljs

```diff
@@ -1,93 +0,0 @@
-(ns status-im.biometric-auth.core
```

Comment:
this makes sense to move this into auth.biometric.cljs but not in multiaccount considering the future changes coming